R/Ostats.R

```diff
@@ -1,15 +1,16 @@
-#' @title Calculate O-statistics (community-level pairwise niche overlap statistics)
```

Comment:
Here I think the clarification in parentheses is still good to have because "O-statistic" is our jargon and it's good to define. We can still call them community-level statistics in the title even if we clarify later on that it need not be a "community" per se

Reply:
done